src/vendor/helpers/__tests__/diveTo-enzyme.spec.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line max-classes-per-file
```

Comment:
use camleCase for file names

Reply:
I have this file from template, camelCase not save this:)